test/visual/visual-constants.ts

```diff
@@ -0,0 +1,10 @@
+export const MICRO_ANIMATION_TIMEOUT = 350;
+export const MICRO_ANIMATION_TIMEOUT_LONG = 500;
+
+export const ELEMENT_PADDING_OFFSET = 25;
+
+export const RESOLUTIONS = [
```

Comment:
Actually, i don't fully understand `RESOLUTIONS` idea. Is it re-invention of [`playwright emulation`](https://playwright.dev/docs/emulation) or something different?